Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM hexpm/elixir:1.15.7-erlang-26.1-debian-bookworm-20230612-slim as base
+
+# install build dependencies
+# --allow-releaseinfo-change allows to pull from 'oldstable'
+RUN apt-get update --allow-releaseinfo-change -y && \
+    apt-get install -y \
+    build-essential \
+    git \
+    openssl \
+    ca-certificates \
+    inotify-tools && \
+    apt-get clean && \
+    rm -f /var/lib/apt/lists/*_*
+
+# Install hex
+RUN mix local.hex --force && \
+    mix local.rebar --force && \
+    mix hex.info
+
+WORKDIR /src
+
+FROM base as deps
+# Cache elixir deps
+ADD mix.exs mix.lock ./
+RUN mix do deps.get, deps.compile
+
+FROM deps as builder
+# Add all the rest
+ADD . .
+
+# ------------------------
```

Comment:
Remove (or put the separator between every stage, for consistency)

Reply:
The separator divides _prod_ between _dev_. No problem to add it to each layer

Reply:
Maybe a comment could be more significant than a separator in this case

Reply:
I've just added both, one doesn't seem to be enough